### PR TITLE
[proposal] Streamline, simplify README

### PR DESCRIPTION
**Pull request by holyjak**
I felt overwhelmed and confused reading the README and trying to get started. There were too many pieces, too many options. I wrote this as an example how it could be simplified so that a newcomer can find her way to running and interacting with the app more quickly.

It would be even better if `clj -A:dev ...` actually started a background nREPL server in addition to the interactive REPL itself, since you could then connect to it from Cursive/... and use the Quick Start as-is to start coding, similarly to what `lein repl` does.

BEWARE: If you like it, at can be merged of course but it was primarily intended to kick of a discussion because I suspect you will have significant input on this...

### CODE REVIEW

**Comment by awkay** [comment]
Hey there. Thanks for thinking through this.  I do not have a lot of extra time for making changes primarily because I don't have the time to carefully read them and such.  That tends to make me lean conservatively and leave "well enough" alone.  That said, I really don't want to discourage contributions, because they are the key to making this a more sustainable effort.

Ideally this would be something like a lein template where the user could choose which things they want, so it could emit a very minimal thing for absolute beginners, and the advanced user that wants a production-ready template (which is more what this is) could get that as well.

I don't have time to maintain both, and definitely don't have time to maintain a programmatic template. 

So, I don't mind the changes you've suggested.  If you really feel they will help a beginner, then that's great.

Starting some kind of lein-based template (which could be based on the old Fulcro 2 lein template) is also something I'm interested in having, but have no time to actually do at the moment.

File: README.adoc [hunk 1]
```diff
@@ -9,61 +9,70 @@ endif::[]
...
+yarn install # or: npm install
 
-#### Refreshing tests in the browser ####
-# CLJS Tests:
-npm run client/test
-# Visit http://localhost:8022
-
-#### full-stack development ####
-# Start the cljs compiler for the main target
-npm run client/main
```
Comment (awkay) [inline README.adoc:46]: I'm personally not a fan of npm run...this was someone else's suggestion.  I'd much rather ppl just see the raw command.

**Comment by awkay** [comment]
In terms of your nREPL comments.  Maybe a link to the YouTube series that shows setting up the run configs in Cursive...so that ppl could see how to get going in IntelliJ?  I think that's ultimately what I'd want as a developer: how do I get it running in an env where I can do real work?

File: README.adoc (hunk 1 above)
Comment (holyjak) [reply README.adoc:46]: I see your point. Changed.

**Comment by holyjak** [comment]
> In terms of your nREPL comments. Maybe a link to the YouTube series that shows setting up the run configs in Cursive...so that ppl could see how to get going in IntelliJ? I think that's ultimately what I'd want as a developer: how do I get it running in an env where I can do real work?

I couldn't find such a video but I linked to Cursive docs.

**Comment by awkay** [comment]
Cool, thanks

**Pull request merged**